handlers/alarms-handler/src/index.ts

```diff
@@ -42,6 +41,44 @@ export const handler = async (event: SQSEvent): Promise<void> => {
 	}
 };
 
+export async function getChatMessages(
+	record: SQSRecord,
+	alarmMappings: AlarmMappings,
+): Promise<{ webhookUrls: string[]; text: string } | undefined> {
+	console.log('sqsRecord', record);
+
+	const { Message, MessageAttributes } = JSON.parse(
+		record.body,
+	) as SNSEventRecord['Sns'];
+
+	console.log('snsEvent', Message, MessageAttributes);
+
+	const parsedMessage = attemptToParseMessageString({
+		messageString: Message,
+	});
+
+	console.log('parsedMessage', parsedMessage);
+
+	let message;
+	if (parsedMessage) {
+		message = await handleCloudWatchAlarmMessage({ message: parsedMessage });
+	} else {
+		message = handleSnsPublishMessage({
+			message: Message,
+			messageAttributes: MessageAttributes,
+		});
+	}
+
+	if (message) {
+		const teams = alarmMappings.getTeams(message.app);
+		console.log('sending message to teams', teams);
```

Comment:
If we keep this log line, what about `mapping message to teams` since we aren't actually sending it here?

```suggestion
		console.log('mapping message to teams', teams);
```

Reply:
thanks, went for
`console.log(`app ${message.app} is assigned to teams ${teams.join(', ')}`);`
 in the end